.github/workflows/stop-dev-environment.yml

```
@@ -0,0 +1,33 @@
+      - name: Configure Twingate
+        uses: twingate/github-action@v1
+        with:
+          service-key: ${{ secrets.TWINGATE_SERVICE_KEY_SECRET_NAME }}
+
+      - name: Execute EKS Stop Action
+        uses: ./.github/actions/stop-dev
+        with:
+          aws-region: ${{ secrets.AWS_REGION }}
+          aws-role: ${{ secrets.AWS_ASSUME_ROLE }}
+          eks-cluster: ${{ secrets.EKS_CLUSTER_NAME }}
+          environment-name: 'dev'
```

actionlint (1.7.4)

input "environment-name" is not defined in action "Stop EKS Environment" defined at "./.github/actions/stop-dev". available inputs are "aws-region", "aws-role", "eks-cluster"

(action)